LLama/LLamaExecutorBase.cs

```diff
@@ -209,7 +209,7 @@ protected virtual void HandleRunOutOfContext(int tokensToKeep)
         /// <summary>
         /// Try to reuse the matching prefix from the session file.
         /// </summary>
-        protected virtual void TryReuseMathingPrefix()
```

Comment:
This is _technically_ a breaking prefix. But I'm not too concerned about it in this case tbh.

Reply:
We could mention it in the next release note. :)